styles/_base.scss

```diff
@@ -148,3 +144,8 @@ ul {
   width: 1px;
   white-space: nowrap;
 }
+
+:focus {
```

Comment:
I don't think this will work.

Reply:
I think you're right. I dun goofed.

Reply:
![](https://media3.giphy.com/media/13FrpeVH09Zrb2/source.gif)